lib/saml/errors.rb

```diff
@@ -19,11 +19,14 @@ class UserAttributeError < SAMLError
                           tag: :idme_uuid_missing,
                           message: 'User attributes is missing an ID.me UUID' }.freeze
 
-    def initialize(message:, code:, tag:)
+    attr_reader :identifier
+
+    def initialize(message:, code:, tag:, identifier: nil)
       @code = code
       @tag = tag
       @level = :warning
       @context = {}
+      @identifier = identifier
```

Comment:
Do we need to filter this in sentry?

Reply:
I don't think so; ICN is fair game to log, like ID.me UUID and secid. But we should go through and bring https://github.com/department-of-veterans-affairs/vets-api/pull/4095 to completion.